src/Network/Receive.pm

```diff
@@ -3040,6 +3040,45 @@ sub vending_start {
 	$shopEarned ||= 0;
 }
 
+sub vender_items_list {
+	my ($self, $args) = @_;
+
+	my $item_pack = $self->{vender_items_list_item_pack} || 'V v2 C v C3 a8';
```

Comment:
add `$self->{vender_items_list_item_pack} = 'V v2 C v C3 a8 a25';` to `RagexeRE_2015_05_13a`

Reply:
https://github.com/OpenKore/openkore/pull/2346/files

already added and checked

this pull is only to avoid manual unpacking and manual skip

i removed and updated this
```
my $msg = $args->{RAW_MSG};
	my $msg_size = $args->{RAW_MSG_SIZE};
	my $headerlen;
	my $item_pack = $self->{vender_items_list_item_pack} || 'V v2 C v C3 a8';
	my $item_len = length pack $item_pack;

	# a hack, but the best we can do now
	if ($args->{switch} eq "0133") {
		$headerlen = 8;
	} else { # switch 0800
		$headerlen = 12;
	}
```